[2021-08-04 14:50:19,403] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_A 2021-08-04T14:50:07.958228+00:00 [queued]>
[2021-08-04 14:50:19,420] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_A 2021-08-04T14:50:07.958228+00:00 [queued]>
[2021-08-04 14:50:19,420] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 14:50:19,421] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-04 14:50:19,421] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 14:50:19,445] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_A> on 2021-08-04T14:50:07.958228+00:00
[2021-08-04 14:50:19,450] {standard_task_runner.py:52} INFO - Started process 2679 to run task
[2021-08-04 14:50:19,455] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_A', '2021-08-04T14:50:07.958228+00:00', '--job-id', '3159', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpkt8twvry', '--error-file', '/tmp/tmpuimv67lx']
[2021-08-04 14:50:19,455] {standard_task_runner.py:77} INFO - Job 3159: Subtask training_model_A
[2021-08-04 14:50:19,523] {logging_mixin.py:104} INFO - Running <TaskInstance: my_dag.training_model_A 2021-08-04T14:50:07.958228+00:00 [running]> on host 04479ea62028
[2021-08-04 14:50:19,628] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_A
AIRFLOW_CTX_EXECUTION_DATE=2021-08-04T14:50:07.958228+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-04T14:50:07.958228+00:00
[2021-08-04 14:50:19,629] {python.py:151} INFO - Done. Returned value was: 5
[2021-08-04 14:50:19,679] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_A, execution_date=20210804T145007, start_date=20210804T145019, end_date=20210804T145019
[2021-08-04 14:50:19,720] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-04 14:50:19,750] {local_task_job.py:149} INFO - Task exited with return code 0
